src/checks/non_committal/prompt.py

```diff
@@ -0,0 +1,21 @@
+from typing import List
+from jinja2 import Template
+import json
+from typing import Optional
+
+def detect_noncommittal_response(question: str, answer: str, prompt_path: str):
```

Comment:
Since multiple values are being returned here, is it as a list ? Can we specify its type in the function signature